src/webgpu/api/validation/texture/rg11b10ufloat_renderable.spec.ts

```diff
@@ -16,10 +16,20 @@ sampleCount > 1, iff rg11b10ufloat-renderable feature is enabled.
 Note, the createTexture tests cover these validation cases where this feature is not enabled.
 `
   )
-  .params(u =>
-    u.combine('usage', [0, GPUConst.TextureUsage.RENDER_ATTACHMENT]).combine('sampleCount', [1, 4])
-  )
-  .unimplemented();
+  .params(u => u.combine('sampleCount', [1, 4]))
```

Comment:
As a control case, we should also parameterize over whether the feature is enabled or not. Then, createTexture should succeed "iff" the feature is enabled.

Same comment on all 4 of the tests here.

Reply:
Sorry if I misunderstand something but we decided not to add enable parameter in this test case because non-feature-enabled test is covered by other tests and we added a note about it as comment, didn't we?

Reply:
Sorry, I forgot about that conversation.

I was thinking there's a problem since we've removed the other parameterizations, and now there is no control case at all. However, since this is just a positive validation test (it expects validation *succeeds*), it's fine. Control cases are only strictly necessary for negative validation tests, to make sure the error isn't coming from the wrong place.